test/behaviour/protractor.browserstack.conf.js

```diff
@@ -100,7 +100,11 @@ exports.config = {
 		reporter: 'min'
 	},
 	specs: ['**/*.js'],
-	exclude: ['**/*.conf.js']
+	exclude: ['**/*.conf.js'],
```

Comment:
This was already removed in 2cb90f1b81b0 (but is currently not merged to this branch).